frontend/src/pages/IstioConfigList/IstioConfigListPage.tsx

```diff
@@ -153,27 +156,49 @@ class IstioConfigListPageComponent extends FilterComponent.Component<
   render() {
     return (
       <>
-        <div style={{ backgroundColor: '#fff' }}>
-          <DefaultSecondaryMasthead
-            rightToolbar={<RefreshButtonContainer key={'Refresh'} handleRefresh={this.updateListItems} />}
-            actionsToolbar={<IstioActionsNamespaceDropdown />}
-          />
-        </div>
-        <RenderContent>
-          <VirtualList rows={this.state.listItems}>
-            <StatefulFilters
-              initialFilters={IstioConfigListFilters.availableFilters}
-              onFilterChange={this.onFilterChange}
+        {this.props.istioAPIEnabled === true && (
+          <>
+          <div style={{ backgroundColor: '#fff' }}>
+            <DefaultSecondaryMasthead
+              rightToolbar={<RefreshButtonContainer key={'Refresh'} handleRefresh={this.updateListItems} />}
+              actionsToolbar={<IstioActionsNamespaceDropdown />}
             />
-          </VirtualList>
-        </RenderContent>
+          </div>
+          <RenderContent>
+            <VirtualList rows={this.state.listItems}>
+              <StatefulFilters
+                initialFilters={IstioConfigListFilters.availableFilters}
+                onFilterChange={this.onFilterChange}
+              />
+            </VirtualList>
+          </RenderContent>
+          </>)}
+        {this.props.istioAPIEnabled === false && (
+          <>
+            <div style={{ backgroundColor: '#fff' }}>
+              <DefaultSecondaryMasthead
+              />
+            </div>
+            <RenderContent>
+              <div style={{ padding: '10px' }}>
+                <WarningTriangleIcon color={PFColors.Warning} style={{marginRight: '5px'}} /> Istio API is disabled and Istio Config is read only.
```

Comment:
Why is editing istio config disabled when the registry is disabled?

It looks like you can still use the wizards to create/update/delete istio config.